datalake/src/main/java/com/sequenceiq/datalake/flow/detach/SdxDetachActions.java

```diff
@@ -301,8 +301,11 @@ protected void doExecute(SdxContext context, SdxDetachFailedEvent payload, Map<O
                 Exception exception = payload.getException();
                 LOGGER.error("Detaching SDX cluster with ID {} and name {} failed with error {}.",
                         payload.getResourceId(), payload.getSdxName(), exception.getMessage(), exception);
-                String statusReason = Optional.ofNullable(exception.getMessage()).orElse("SDX detach failed");
-                sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.STOPPED, statusReason, payload.getResourceId());
+                String statusReason = Optional.ofNullable(exception.getMessage()).orElse("null exception");
```

Comment:
`null exception` is not a customer friendly message, we need something which can be undestandle from customer point of view
if there is a possible null in the code which we should handle, then we should handle it or log it if it is unexpected

Reply:
Completely fair, I will update this.

Reply:
So I am changing this to just say `unknown error` but don't find that to be vastly different. Note that this field is used in the following line so the overall message in this scenario will be:
`Sdx detach failed due to: unknown error`

Let me know if you have any other ideas for a more user friendly message. I do not expect null exception messages to happen a lot and especially not for any of the most common errors.

Reply:
this should be good if the message is null, thx!